docs/troubleshooting.md

```diff
@@ -130,3 +130,183 @@ agent:
 ```
 
 Similar can be applied to any other failing exporter.
+
+## Possible problems with Kubernetes and container runtimes
+
+A Kubernetes cluster using an incompatible container runtime could experience:
```

Comment:
I assumed the incompatibility was with our chart so may be helpful to stress that it's a cluster-wide issue?

```suggestion
A Kubernetes cluster using an incompatible container runtime for its version or configuration could experience:
```

Reply:
True. Done.